docs/2.8.0/docs/daml-finance/instruments/structured-product.rst

```diff
@@ -0,0 +1,78 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+How to use the Structured Product Instrument package
+####################################################
+
+To follow the script used in this tutorial, you can
+`clone the Daml Finance repository <https://github.com/digital-asset/daml-finance>`_. In particular,
+the Structured Product test folder
+`Instrument/StructuredProduct/Test/ <https://github.com/digital-asset/daml-finance/blob/main/src/test/daml/Daml/Finance/Instrument/StructuredProduct/Test/>`_
+is the starting point of this tutorial.
+
+How To Create a Structured Product Instrument
```

Comment:
I would align with the other pages, such as 

```suggestion
How to create a Structured Product Instrument
```

Reply:
Not sure what the correct standard is, so I opened this: https://github.com/digital-asset/daml-finance/issues/1004
(covers other aspects as well).

Reply:
thanks 👍 shall we align the format anyway and then let them review?

Reply:
I think it is better to wait and do it right once.